core/__init__.py

```diff
@@ -308,7 +308,7 @@ def forcePost(f: Callable) -> Callable:
 	return f
 
 
-def exposed(f: Callable) -> Callable:
+def exposed(f: Union[Callable, dict]) -> Callable:
```

Comment:
@sveneberth can you please shortly describe this change?

Reply:
The `exposed` method is usable in the common way as decorator (which will be called with your method as `Callable` argument). Or it's callable with a `dict` as argument to provide a translated/seo-friendly name for this method (and generates a new method (`exposeWithTranslations`) to wrap your method).

Example: 
```py
class MyModule(List):
	@exposed({
		"de": "aktuelle-Uhrzeit",
		"en": "current-time",
	})
	def currenttime(self):
		return f"It's {datetime.datetime.now().isoformat(' ', 'seconds')}"

# On the German site:
# GET /de/mymodule/aktuelle-Uhrzeit

# On the English site:
# GET /en/mymodule/current-time
```

I want to write a detailed site/example in our docs about the SEO-features in the core - Maybe on the next OpenSourceFriday :)

Reply:
Yes, it would be nice to have more parameters, e.g. to restrict access for this method by access levels (as far as i remember, we want a logic for this which is non-public by default (only root-users) and additional users have to be explicitly permitted. 

But this doesn't really have to do with the PR... Maybe we should create an issue to collect some ideas or discuss this. :)

Reply:
> Maybe we should create an issue to collect some ideas or discuss this. :)

I've opened #412 on this, please add further suggestions there.